### PR TITLE
Interpolate environment variables

### LINKED ISSUE #31: Some messaging about overriding values
I think it would be good, if the user picks `aws` provider, to show a message saying

Environment is not configured properly, please set

- CONTROL_PLANE_INSTANCE_TYPE
- SSH_KEY_NAME

And then perhaps even defaults to copy paste like

```
export CONTROL_PLANE_INSTANCE_TYPE=t2.medium
export SSH_KEY_NAME=default
```

and then a way to avoid this error such as --allow-unset-env-vars (-f?)

### PR DESCRIPTION
**Pull request by chuckha**
This is a little bit smarter and will show you defaults to set.

If you do not want defaults you can override the behavior with a flag
and only interpolate environment variables that *are* defined while
leaving the placeholders intact

Signed-off-by: Chuck Ha <chuckh@vmware.com>

fixes #31 

WDYT @ashish-amarnath ? I kinda like how this UI feels

### CODE REVIEW

File: cmd/generate.go [hunk 1]
```diff
@@ -151,7 +171,15 @@ func runGenerateCommand(opts generateOptions, stdout io.Writer) error {
 		if err != nil {
 			return err
 		}
-		fmt.Fprintln(stdout, string(b))
+		fmt.Fprintln(stdout, os.Expand(string(b), envFn))
```
Comment (ashish-amarnath) [inline cmd/generate.go]: TIL `os.Expand` Thank you 🙏

Comment (chuckha) [reply cmd/generate.go]: haha I also learned about it today!

**Comment by ashish-amarnath** [comment]
Yeah, the UI looks great!
I was thinking about a config file instead of env vars, thinking that the env vars maybe a little too invasive into the developers' environment. But we could always add another source of these variables.
Thanks for getting to this 😄

**Comment by chuckha** [comment]
Yeah, I'm starting to think a config would be really useful...i mean, i do love a bash one liner without a file, but a config file would be really nice for edits/VCS...plus we could supply some example configs that would have instant utility

**Comment by chuckha** [comment]
@ashish-amarnath anything else before we merge?

**Pull request merged**